src/mixins/filing-template-mixin.ts

```diff
@@ -196,6 +198,53 @@ export default class FilingTemplateMixin extends DateMixin {
       }
     }
 
+    if (this.isCoopCorrectionFiling) {
+      // Parties and offices aren't required for Coop corrections.
+      delete filing.correction.parties
+      delete filing.correction.offices
+      filing.correction = {
+        ...filing.correction,
+        resolution: this.getSpecialResolution.resolution,
+        resolutionDate: this.getSpecialResolution.resolutionDate,
+        signingDate: this.getSpecialResolution.signingDate,
+        signatory: this.getSpecialResolution.signatory
+      }
+
+      // Apply Court Order ONLY when it is required and applied
+      if (this.getHasPlanOfArrangement || this.getFileNumber) {
+        filing.correction.courtOrder = {
+          fileNumber: this.getFileNumber,
+          effectOfOrder: this.getHasPlanOfArrangement ? EffectOfOrders.PLAN_OF_ARRANGEMENT : null,
+          hasPlanOfArrangement: this.getHasPlanOfArrangement
+        }
+      }
+
+      if (this.hasAssociationTypeChanged || this.hasSpecialResolutionMemorandumChanged ||
+          this.hasSpecialResolutionRulesChanged) {
```

Comment:
Instead of wrapping, in this case, I'd prefer to see:
```
if (
  a ||
  b ||
  c
} {
  ...
}
```

(Easier to read/review/understand. Thanks!)

Reply:
Is there an ESLINT rule we could apply her perhaps?

Reply:
Maybe one of these? https://eslint.org/docs/latest/rules/#layout--formatting

If there isn't one already, we could create a custom rule.

Reply:
No luck, spent an hour looking for one.

Reply:
Darn.

Thanks for the code change. My goal is easy-to-review (and thus read/understand) code. "If" conditions that fit on 1 line are fine, otherwise breaking them up as you've done is really clean IMO.